src/crimson/os/seastore/seastore.cc

```diff
@@ -1,5 +1,5 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
-// vim: ts=8 sw=2 smarttab
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
+// vim: ts=8 sw=2 smarttab expandtab
```

Comment:
You've changed this rule, but not actually updated the whitespace elsewhere in this file.  If you really want to do this, please do it in its own PR and update all of the files in seastore/ at the same time.